Basic-Car-Maintenance/Shared/Localizable.xcstrings

```diff
@@ -148,13 +151,13 @@
     "Failed To Add Vehicle. Unknown Error." : {
 
     },
-    "Failed To Delete Vehicle" : {
+    "Failed To Delete Event" : {
 
     },
-    "Failed To Delete Vehicle\nDetails:%@" : {
+    "Failed To Delete Vehicle" : {
 
     },
-    "Failed To Delete Event" : {
+    "Failed To Delete Vehicle\nDetails:%@" : {
 
```

Comment:
I want to merge this, but this file is one that I'm worried about. I wouldn't expect any of these values to have changed (with the scope of your task), did you have to resolve a merge conflict here?

Reply:
I just aw this in another PR, it must just be how the Localizable.xcstrings file works and moves stuff around